cclib/parser_properties/mosyms.py

```diff
@@ -0,0 +1,79 @@
+from cclib.parser_properties import utils
+from cclib.parser_properties.base_parser import base_parser
+
+import numpy as np
+
+
+class mosyms(base_parser):
+    """
+    Docstring? Units?
+    """
+
+    known_codes = ["gaussian"]
+
+    @staticmethod
+    def normalisesym(label):
```

Comment:
`normalisesym` -> `gaussian_normalizesym`? So we don't call it from other programs